src/modules/contactUs/contact-us.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Body, Controller, HttpStatus, Post, UsePipes, ValidationPipe } from '@nestjs/common';
+import { EmailService } from '../email/email.service';
+import { ContactUsDto } from './dto/contact-us.dto';
+import { skipAuth } from '../../helpers/skipAuth';
+import { ApiBody, ApiResponse } from '@nestjs/swagger';
+import { ContactUs } from './entities/contact-us.entity';
+
+@skipAuth()
+@Controller('/contact-us')
+export class ContactController {
+  constructor(private readonly contactService: EmailService) {}
+
+  @Post()
+  @UsePipes(new ValidationPipe({ whitelist: true, forbidNonWhitelisted: true }))
```

Comment:
This can be removed. Its been applied globally in `app.module.ts`

Reply:
done @CFCIfe